docs/display.rst

```diff
@@ -0,0 +1,54 @@
+Display
+*******
+
+.. py:module:: microbit.display
+
+This module controls the 5×5 LED display on the front of your board. It can
+be used to display images, animations and even text.
+
+Functions
+=========
+
+.. py:function:: get_pixel(x, y)
+
+    Return the brightness of the LED at column ``x`` and row ``y``.
+
+
+.. py:function:: set_pixel(x, y)
+
+    Set the brightness of the LED at column ``x`` and row ``y``.
```

Comment:
Argh, forgot ``value``. Will fix that when filling the rest of missing information.